src/Files.App/Views/Properties/MainPropertiesPage.xaml.cs

```diff
@@ -388,6 +403,13 @@ public bool IsSelected
 			get => _isSelected;
 			set => SetProperty(ref _isSelected, value);
 		}
+
+		private bool _isCompacted;
+		public bool IsCompacted
```

Comment:
What is this property for?

Reply:
The labels such as General, Customisation will not be collapsed automatically when NavigationView mode is LeftCompact. To fix that, detect compact mode and collapse text with this property.

Reply:
Where is it set?

Reply:
Shouldn't it be `isCompact`?